rubygem-foreman_templates/rubygem-foreman_templates.spec

```diff
@@ -35,6 +35,10 @@ Requires: %{?scl_prefix}rubygem(git)
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
 BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildRequires: %{?scl_prefix_ruby}rubygems
+BuildRequires: %{?scl_prefix}rubygem(diffy)
+BuildRequires: %{?scl_prefix}rubygem(git)
```

Comment:
After I thought more about this, it's actually a bug in foreman_templates. It's usable without git. I'll add this as a dependency for now but I opened http://projects.theforeman.org/issues/19381 to ensure the git is not required and then we can remove it from here. If the issue is in git gem loading itself then I'll look if it can be fixed there.